messages/author_converter.py

```diff
@@ -0,0 +1,16 @@
+def convert_entity_into_to(author):
```

Comment:
This could probably be put inside a base class `JSONConvertible` like mentioned above, and could use `Schematics` for the conversion.